docs/index.rst

```diff
@@ -48,41 +48,50 @@
+      * Multi-domain: designed to interact across physical and virtual domains, such as WAN, edge, access network, NFV, cloud, containers, and datacenter.
+      * Holistic: A single, unifying automation solution, providing service orchestration, network orchestration, NFV orchestration (NFVO), as well as generic VNF management (gVNFM), cloud orchestration and configuration management. No other automation tools required.
+      * Full lifecyle: Manage advanced service lifecycle, covering creation, on-boarding, provisioning, modification, scaling, upgrading and decommissioning.
 
     * **Intent-based programmability**: Inmanta optimizes service development and maintenance for telecom operators and service providers through its unifying, model-driven methodology for intent-based orchestration.
 
-    * Inmanta's powerful domain-specific language (DSL) simplifies service creation and management, and is based on infrastructure as code (IaC) principles to provide a unified way to automate multi-domain and multi-vendor services. The embedded DSL enables the development of modular building blocks that make abstraction of low-level details, enabling re-usability across use cases.
-    * Inmanta's intent-based programmability provides out-of-the-box self-healing, safe roll-back, detailed dry run and seamless service upgrades for enhanced stability and resilience.
+      * Inmanta's powerful domain-specific language (DSL) simplifies service creation and management, and is based on infrastructure as code (IaC) principles to provide a unified way to automate multi-domain and multi-vendor services. The embedded DSL enables the development of modular building blocks that make abstraction of low-level details, enabling re-usability across use cases.
+      * Inmanta's intent-based programmability provides out-of-the-box self-healing, safe roll-back, detailed dry run and seamless service upgrades for enhanced stability and resilience.
 
     * **Vendor agnostic**: Inmanta Service Orchestrator is truly open and vendor agnostic for all network layers, domains and OSS/BSS. Service providers can integrate with 3rd party solutions as well as a wide range of open-source technologies to build a best-in-class, all-encompassing solution.
 
-    * Interoperability through pluggable adapters and open APIs
-    * API-ification of orchestrated services to easily plug services into the OSS/BSS environment
-    * Support for brownfield environments by fine-grained roll-out
+      * Interoperability through pluggable adapters and open APIs
+      * API-ification of orchestrated services to easily plug services into the OSS/BSS environment
+      * Support for brownfield environments by fine-grained roll-out
 
     The Inmanta Service Orchestrator product is based on mature technology backed by 15+ years of research and interaction with companies offering telecom and cloud services.
 
 
 .. toctree::
-    :maxdepth: 1
+    :maxdepth: 3
```

Comment:
The ToC looks very polluted to me with a depth of 3. I personally don't see any value in it. I'd ask Bart when he's back to be sure (I wouldn't block the merge on it though, just make sure to ask if he has an opinion when he's back). @arnaudsjs do you have an opinion on this?

Reply:
I prefer it with 1 to.

Reply:
I agree that 1 is better than 3.